tests/testthat/test-redirection.R

```diff
@@ -0,0 +1,14 @@
+test_that("Get redirection works", {
```

Comment:
Use some real websites as well for the test. I suggest

neverssl.com for no redirection. google.com for redirection

Reply:
I thought maybe relying on external websites for tests was a bit volatile but sure, can replace this test with real websites.